### ISSUE TITLE
core_validation layer gives obscure layer names

### ISSUE THREAD

**Issue by krOoze** [KhronosGroup/Vulkan-LoaderAndValidationLayers#1724]
core_validation layer uses obscure `pLayerPrefix` names like "DS", "DL", "MEM", ...

A) Does anybody know what these mean?

B) `README.md` where I looked for answer links to `layers/vk_validation_layer_details.md`. That document is non-existent.

C) It should be changed to something obvious. And it should at least containt the name of the source layer.

**Comment by tobine** [comment]
Yes, these are a bit outdated and come from the time when all layers were separate.
DS = Draw State
DL = Device Limits
MEM = Memory

We'll soon have string-based unique IDs that may make sense here in addition to unique enum. Those may make sense in this field.

I'll discuss this with LunarG and we'll work out something more sane.
